docs/docs/04-alternative-model-providers.md

```diff
@@ -0,0 +1,56 @@
+# Alternative Model Providers
+
+
+## Usage
+
+GPTScript can be used against alternative models that expose an OpenAI compatible API or have a provider shim available.
+
+To use a model with an OpenAI compatible API:
+
+```gptscript
+model: mistral-large-latest from https://api.mistral.ai/v1
+
+Say hello world
+```
+
+To use a model that requires a provider shim:
+
+```gptscript
+model: claude-3-haiku-20240307 from github.com/gptscript-ai/anthropic-provider
+
+Say hello world
+```
+
+### Authentication
+
+For OpenAI compatible providers, GPTScript will look for an API key to be configured with the prefix `GPTSCRIPT_PROVIDER_`, the base domain converted to environment variable format, and a suffix of `_API_KEY`.
+As an example if you are using `mistral-large-latest from https://api.mistral.ai/v1`, the environment variable would be `GPTSCRIPT_PROVIDER_API_MISTRAL_AI_API_KEY`
+
+Each provider shim has different requirements for authentication. Please check the readme for the provider you are trying to use.
+
+## Available Model Providers
+
+The following shims are currently available:
+* github.com/gptscript-ai/bedrock-provider
+* github.com/gptscript-ai/anthropic-provider
```

Comment:
lets link out since we're telling them to look at the readme
```suggestion
* [github.com/gptscript-ai/anthropic-provider](https://github.com/gptscript-ai/anthropic-provider)
```